src/streamlink_cli/utils/player.py

```diff
@@ -36,3 +38,20 @@ def find_default_player():
     if path:
         # Quote command because it can contain space
         return subprocess.list2cmdline([path])
+
+def sanitizeTitle(wholeTitle):
+    if wholeTitle is None:
+        wholeTitle = DEFAULT_FORMAT_ARGUMENTS["title"]
+    
+    if not is_win32:
+        if is_py2:
+            title = shlex_quote(wholeTitle.encode('utf8'))
+        elif is_py3:
+            title = shlex_quote(wholeTitle)
+    else:
+        if is_py2:
+            title = subprocess.list2cmdline([wholeTitle.encode('utf8')])
+        elif is_py3:
+            title = subprocess.list2cmdline([wholeTitle])
+    title = title.replace("$","$$")
```

Comment:
It just turns out to be necessary on both Windows and Posix. I found out that if you have 2 $ in a row, it replaces it with 1 $, since $ is a special character.